packages/core-js/internals/regexp-exec.js

```diff
@@ -18,24 +19,41 @@ var UPDATES_LAST_INDEX_WRONG = (function () {
   return re1.lastIndex !== 0 || re2.lastIndex !== 0;
 })();
 
+var UNSUPPORTED_Y = stickyHelpers.UNSUPPORTED_Y || stickyHelpers.BROKEN_CARET;
+
 // nonparticipating capturing group, copied from es5-shim's String#split patch.
 var NPCG_INCLUDED = /()??/.exec('')[1] !== undefined;
 
-var PATCH = UPDATES_LAST_INDEX_WRONG || NPCG_INCLUDED;
+var PATCH = UPDATES_LAST_INDEX_WRONG || NPCG_INCLUDED || UNSUPPORTED_Y;
 
 if (PATCH) {
   patchedExec = function exec(str) {
     var re = this;
     var lastIndex, reCopy, match, i;
+    var sticky = UNSUPPORTED_Y && re.sticky;
+
+    if (sticky) {
+      var flags = (re.ignoreCase ? 'i' : '') +
+                  (re.multiline ? 'm' : '') +
+                  (re.unicode ? 'u' : '') +
```

Comment:
Seems also here.

Reply:
I don't understand how this can throw: `re.unicode` will always be false in old envs (e.g. IE8)